File: web3rpc/rpc-specs/code-samples/python/klay/block/getRewards.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockTag = "latest"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_rewards(blockTag)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getRewards.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumberOrTag = "latest";
  provider.klay
    .getRewards(blockNumberOrTag, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
